config/routes.rb

```diff
@@ -276,6 +276,8 @@ def paginate_url(url, params)
       post '/dde/patients/merge', to: 'dde#merge_patients'
       get '/dde/patients/remaining_npids', to: 'dde#remaining_npids'
       get '/rollback/merge_history', to: 'rollback#merge_history'
+      post '/dde/patients/duplicates', to: 'dde#duplicates_finder'
```

Comment:
I thinking let us also have this as a background job which sometime off working hours